ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -61,23 +55,11 @@ public class BottomSheetController: UIViewController {
         }
     }
 
-    /// Indicates if `preferredContentSize` of `contentViewController` should be respected.
-    /// Regardless of the value, the expanded height is limited by the height of `BottomSheetController`'s root view.
-    @objc open var respectsPreferredContentSize: Bool = false {
```

Comment:
This is going away for now because we don't support content UIViewControllers anymore. New sizing API (other than the current `expandedHeightFraction`) will be added together with multiple expansion states.